spec/fx/adapters/postgres_spec.rb

```diff
@@ -20,6 +20,38 @@ module Fx::Adapters
       end
     end
 
+    describe "#update_function" do
+      let(:name) { "test" }
```

Comment:
I don't really rely on `let` anywhere else in the codebase, can you instead inline the 'let's as local variables?

Reply:
The idea behind `let` is to avoid duplicating things across different scenarios.

Even so do you still want to change to local variables?

Reply:
That's right.